cloudy/src/main/kotlin/com/skydoves/cloudy/internals/render/RenderScriptToolkit.kt

```diff
@@ -315,10 +316,10 @@ internal fun vectorSize(bitmap: Bitmap): Int {
   }
 }
 
-internal fun iterativeBlur(
+internal suspend fun iterativeBlur(
   androidBitmap: Bitmap,
   radius: Int
-): Bitmap? {
+): Bitmap? = coroutineScope {
```

Comment:
Unsolicited review: 

`coroutineScope {  }` is redundant. You are not launching any other coroutines there. `suspend` is enough. 

Maybe if you have launched each blur region in separate coroutine then it would have been justified. Like

```
coroutineScope  { 
  for (i in 0 until iterate) {
     launch { 
          bitmap = RenderScriptToolkit.blur(
            inputBitmap = bitmap,
            radius = 25
          )
       }
  }
  
  }